.github/workflows/python.yml

```diff
@@ -1,29 +0,0 @@
-name: python
-
-on:
-  pull_request:
-  push:
-    branches: [master, develop]
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        python-version: [3.7, 3.8]
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install -r requirements.txt
-    - name: Test with pytest
-      run: |
-        # add to PYTHONPATH so imports work in test files
-        export PYTHONPATH="${PYTHONPATH}:${pwd}"
-        pip install pytest
-        pytest
```

Comment:
I assume you use something other than pytest so ok with deleting

Reply:
It's just part of `make test` now